static/css/restyle.less

```diff
@@ -1030,3 +1038,81 @@ button.good {
     border-radius: 0;
   }
 }
+
+// Unified theme view
+.personas-featured .persona-list-3col {
+  border: 0;
+  min-height: 125px;
+}
+
+.persona-list-3col {
+  li {
+    border: 0;
+    display: inline;
+    overflow: hidden;
+    padding: .6em 0 0;
+    position: relative;
+  }
+
+  .persona.hovercard {
+    margin-right: 14px;
+    width: 32%;
+
+    a {
+      padding: 0;
+    }
+
+    h3 {
+      max-width: 200px;
+    }
+  }
+}
+
+.category-landing .addons-column ul.personas {
+  padding: 0;
+}
+
+.category-landing .addons-column ul li {
+  overflow: visible;
+
+  .persona a {
+    padding: 0;
+  }
+}
+
+.persona.hovercard {
+  max-width: 200px;
+
+  &:hover {
+    // This decimal point seems nit-picky, but at least on my retina Mac
+    // it prevents the item below from shifting around.
+    margin-bottom: -22.5px;
+    max-height: 105px;
+  }
+
+  h3 {
+    font-size: 14px;
+    padding: 3px 0 0;
+  }
+
+  .persona-preview {
+    float: none;
+  }
+}
+
+// This is used to override JS that tries to mess with these styles :'(
```

Comment:
I'll try it, but that may not work. This was just easier, but frustratingly the hovercard styles are a mix of `:hover` and a class that gets applied on hover named `persona-hover`. :cry: 

I'll try it now though...